moduleroot/Rakefile

```diff
@@ -7,3 +7,27 @@ PuppetLint.configuration.fail_on_warnings = true
 PuppetLint.configuration.send('<%= check %>')
 <% end -%>
 PuppetLint.configuration.ignore_paths = ["spec/**/*.pp", "pkg/**/*.pp"]
+
+desc 'Generate pooler nodesets'
+task :gen_nodeset do
```

Comment:
Mulling over this, I wonder if it wouldn't be better suited to hoock this up directly to https://github.com/puppetlabs/puppetlabs_spec_helper/blob/master/lib/puppetlabs_spec_helper/rake_tasks.rb#L14